src/main/java/org/broadinstitute/consent/http/db/ElectionDAO.java

```diff
@@ -195,6 +197,21 @@ void bulkCancelOpenElectionByReferenceIdAndType(@Bind("electionType") String ele
     @UseRowMapper(SimpleElectionMapper.class)
     List<Election> findLastElectionsByReferenceIdsAndType(@BindList("referenceIds") List<String> referenceIds, @Bind("type") String type);
 
+    @SqlQuery("select distinct e.electionId,  e.datasetId, v.vote finalVote, e.status, e.createDate, e.referenceId, v.rationale finalRationale, " +
+            "v.createDate finalVoteDate, e.lastUpdate, e.finalAccessVote, e.electionType, e.dataUseLetter, e.dulName, e.archived, e.version  from election e " +
+            "inner join (select referenceId, MAX(createDate) maxDate from election e where lower(e.status) = lower(:status) group by referenceId) " +
```

Comment:
I think this query might have the same issue as the MetricsDAO where it tried to find the last elections for `Access` and `RP` by date. The problem was that for some elections the `RP` election was made after the `Access` election  (around a 200ms difference), resulting in the query returning only the final `RP` votes.

Just to be safe, try to find an election on your local db where the RP and Access Elections have different timestamps but the same referenceId and see if this query works correctly.

Reply:
Wait, forget what I said, there's a type parameter, so this query is okay

Reply:
This is the original functionality, so even if it's not 100% correct, it will restore what we had prior to removing it. If this doesn't work as expected, I think we need to tackle that as a separate task.